### PR TITLE
Rails 5 enum

### PR DESCRIPTION
**Pull request by robinbortlik**
fix value of `enum`columns in rails 4.1 and 5.

### CODE REVIEW

File: lib/validates_overlap/overlap_validator.rb [hunk 1]
```diff
@@ -168,14 +168,28 @@ def value_attribute_name(attr_name)
...
+      value
     end
   end
 
+  def is_enum_attribute?(record, attr_name)
+    implement_enum? && record.class.defined_enums[attr_name].present?
+  end
+
+  def implement_enum?
+    (ActiveRecord::VERSION::MAJOR > 5) || (ActiveRecord::VERSION::MAJOR > 4 && ActiveRecord::VERSION::MINOR > 1)
```
Comment (stricte) [inline lib/validates_overlap/overlap_validator.rb]: Should be `ActiveRecord::VERSION::MAJOR >= 5`

Comment (robinbortlik) [inline lib/validates_overlap/overlap_validator.rb]: @stricte yeah, looks like. How did you find this issue?

Comment (stricte) [inline lib/validates_overlap/overlap_validator.rb]: Simple code review. Im interested in enum support.

Comment (robinbortlik) [inline lib/validates_overlap/overlap_validator.rb]: @stricte thank you very much for finding the bug. I have fixed it. I know it took a long for such small thing, but I was quite busy. :) Anyway, thank you again :)

**Pull request merged**